crates/registry/src/package_distribution.rs

```diff
@@ -3,7 +3,7 @@ use serde::{Deserialize, Serialize};
 #[derive(Serialize, Deserialize, Debug, Default, Clone, Eq)]
 #[serde(rename_all = "camelCase")]
 pub struct PackageDistribution {
-    pub integrity: String,
+    pub integrity: String, // TODO: why fetching typescript cause error here?
```

Comment:
Will we merge the PR with TODO items? It is probably best to open issues instead of adding TODO comments.

Reply:
I personally prefer code todos as well as issues. It makes it easier to navigate through the codebase.

Reply:
https://github.com/pnpm/pacquet/issues/92